doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+
+For each tool listed below, we have indicated its specific location within 
+plutus-apps, provided a short description, indicated its dependencies, and given 
+a brief statement about its state of development to give you some context. 
+
+1. Plutus script utils
+--------------------------
+
+`Plutus script utils <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-script-utils>`_ 
+is a utility library for helping users write Plutus scripts that are to be used on-chain. 
+Currently, this is a “kitchen sink” library containing code that was moved out of 
+plutus-ledger-api. 
+
+It provides: 
+
+   * hashing functions for Datums, Redeemers and Plutus scripts for any Plutus language version. 
+   * functionality for wrapping the untyped Plutus script with a typed interface. 
+   * utility functions for working with the ScriptContext of a Plutus Script. 
+
++--------------+---------------------------------------------------------------+
+| Plutus Script Utils                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-script-utils``                           |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress with an unstable interface.                  |
++--------------+---------------------------------------------------------------+
+
+2. Marconi
+--------------------------
+
+`Marconi <https://github.com/input-output-hk/plutus-apps/tree/main/marconi>`_ is an 
+application for indexing useful data from the Cardano blockchain in a scalable way. 
+
++--------------+---------------------------------------------------------------+
+| Marconi                                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/marconi``                                       |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | In a very early alpha version. Has only a few indexers and    |
+|              | the final architecture is currently being designed.           |
++--------------+---------------------------------------------------------------+
+
+3. Plutus ledger
+--------------------------
+
+`Plutus ledger <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger>`_ 
+is a "kitchen sink" set of components containing data types and functions that 
+complement `cardano-ledger <https://github.com/input-output-hk/cardano-ledger>`_ 
```

Comment:
I think kitchen sink here means a set of transitional types that simplify the cardano-api types. This library is being phased out in favor of the real cardano-api types?

Reply:
We have replaced many parts with cardano-api types, but we still simplify them a bit, as we only consider the last era. I guess the goal is to build a comprehensive, easy-to-use set of types replicating the current era of cardano-api.